src/libsystemd-network/dhcp-static-lease.c

```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <errno.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#include "alloc-util.h"
+#include "dhcp-internal.h"
+#include "dhcp-server-internal.h"
+#include "memory-util.h"
+#include "strv.h"
+#include "utf8.h"
+
+static sd_dhcp_static_lease* sd_dhcp_static_lease_free(sd_dhcp_static_lease *i) {
+        return mfree(i);
+}
+
+int sd_dhcp_static_lease_new(DHCPClientId *client_id, struct in_addr *address, sd_dhcp_static_lease **ret) {
```

Comment:
`struct in_addr*` → `const struct in_addr*` 

after all this function is not supposed to change the parameter.